src/msg/producer/buffer/buffer.go

```diff
@@ -117,6 +118,7 @@ func newBufferMetrics(
 		messageBuffered:   scope.Gauge("message-buffered"),
 		byteBuffered:      scope.Gauge("byte-buffered"),
 		bufferScanBatch:   instrument.NewTimer(scope, "buffer-scan-batch", opts),
+		bytesAdded:  scope.Counter("bytes-added"),
```

Comment:
Perhaps name this "buffer-byte-added" to match the "byteDropped" naming?

Also can we add a byteConsumed when "IsDroppedOrConsumed" returns true and message is pruned from queue? That way we can tell bytes in -> matches bytes out <- hopefully.

Reply:
added a `buffer-byte-removed` metric